1-org/envs/shared/variables.tf

```diff
@@ -342,7 +342,33 @@ variable "gcp_billing_creator_user" {
 }
 
 variable "gcp_billing_admin_user" {
-  description = "Identity that has billing administrator permissions"
+  description = "Identity that has billing administrator permissions."
   type        = string
   default     = null
 }
+
+variable "essential_contacts" {
+  description = <<EOT
+Essential Contacts configuration object:
+- language: The preferred language for notifications, as a ISO 639-1 language code. See [Supported languages](https://cloud.google.com/resource-manager/docs/managing-notification-contacts#supported-languages) for a list of supported languages.
+- enable_technical_incidents: See [Assigning contacts for outage and disruption notifications](https://cloud.google.com/resource-manager/docs/managing-notification-contacts#technical-incidents-contacts) for conditions to use Technical Incidents.
```

Comment:
Is this because this is a preview feature? I am leaning towards not having this initially then and once we add the comment with supported types users can expand as needed.

Reply:
Technical incidents removed